components/DropdownLight/DropdownLight.jsx

```diff
@@ -236,56 +278,84 @@ const DropdownLight = ({ disabled, items, theme, placeholder, name }) => {
   }, [enterPress]);
 
   return (
-    <Wrapper>
-      <input
-        type="text"
-        hidden
-        name={name}
-        defaultValue={selectedItem}
-        aria-label="selecione uma opção"
-      />
-
-      <Button
-        aria-haspopup="true"
-        aria-label={isOpen ? 'fechar lista de itens' : 'abrir lista de itens'}
-        onClick={handleToggleDropdown}
-        theme={theme}
-        disabled={disabled}
-        ref={buttonRef}
-      >
-        {itemLabel}
-        <ArrowIcon
-          name={isOpen ? 'arrow_drop_up' : 'arrow_drop_down'}
-          theme={theme}
-        />
-      </Button>
-
-      {isOpen && (
-        <SelectionList theme={theme} ref={listOptions}>
-          {items.map((item, index) => (
-            <SelectionListItem
-              role="option"
-              theme={theme}
-              key={item?.value || item}
-              onClick={() => handleOnClickListItem(item)}
-              aria-posinset={index}
-              aria-selected={index === cursor}
-              tabIndex="-1"
-            >
-              {item?.label || item}
-
-              {(selectedItem === item?.value || selectedItem === item) && (
-                <CheckIcon theme={theme} />
-              )}
-            </SelectionListItem>
-          ))}
-        </SelectionList>
-      )}
-    </Wrapper>
+    <>
+      <ComponentWrapper theme={theme} skin={skin}>
+        <InputWrapper ref={wrapperRef}>
+          <InputLabel error={error}>
+            {label}
+            {required && (
+              <RequiredMark aria-label="campo obrigatório" skin={skin}>
+                *
+              </RequiredMark>
+            )}
+          </InputLabel>
+          <InputText
+            type="text"
+            hidden
+            skin={skin}
+            name={name}
+            defaultValue={selectedItem}
+            aria-label="selecione uma opção"
+            required={required}
+          />
+          <Button
+            aria-haspopup="true"
+            aria-label={
+              isOpen ? 'fechar lista de itens' : 'abrir lista de itens'
+            }
+            onClick={handleToggleDropdown}
+            theme={theme}
+            skin={skin}
+            error={error}
+            disabled={disabled}
+            ref={buttonRef}
+            id={id}
+          >
+            {itemLabel}
+            <ArrowIcon name={isOpen ? 'arrow_drop_up' : 'arrow_drop_down'} />
+          </Button>
+        </InputWrapper>
+        {helperText && (
+          <HelperText aria-label="texto auxiliar">{helperText}</HelperText>
+        )}
+        {error && (
+          <InputErrorMessage
+            aria-label="mensagem de erro"
```

Comment:
When placing aria-label on this element, the screen reader ignores all its content and this is not the desired behavior.